frontend/package.json

```diff
@@ -70,38 +70,38 @@
     "@ng-bootstrap/ng-bootstrap": "11.0.1",
     "@ngx-translate/core": "15.0.0",
     "@ngx-translate/http-loader": "8.0.0",
-    "@valtimo/access-control": "12.6.0",
-    "@valtimo/access-control-management": "12.6.0",
+    "@valtimo/access-control": "12.7.0",
```

Comment:
Please use Valtimo 12.6.0.

If you really need an upgrade to 12.7.0, please do so in a different PR because it involves quite a few more changes.